molecule/default/tests/test_default.py

```diff
@@ -4,40 +4,31 @@
 import os
 
 # Third-Party Libraries
-import pytest
 import testinfra.utils.ansible_runner
 
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
     os.environ["MOLECULE_INVENTORY_FILE"]
 ).get_hosts("all")
 
 
-@pytest.mark.parametrize("pkg", ["python3-pip", "python3-dev"])
-def test_python_debian(host, pkg):
+def test_pip(host):
     """Test that the appropriate packages were installed."""
-    if host.system_info.distribution == "debian" and host.system_info.release != "9.12":
-        assert host.package(pkg).is_installed
-
-
-@pytest.mark.parametrize(
-    "pkg", ["python-pip", "python-dev", "python3-pip", "python3-dev"]
-)
-def test_python_debian_9(host, pkg):
-    """Test that the appropriate packages were installed.
-
-    We treat Debian 9 as a special case because the CyHy AMIs (and
-    only the CyHy AMIs) are built on it.  Therefore it requires
-    python2.
```

Comment:
Are we intentionally dropping documentation about why we are installing python2?  I think it's helpful to keep those explanatory comments somewhere in here, even more so now that we are adding Amazon Linux to the mix.

Reply:
Please see commit 67e2b06.

Reply:
Perfect, thanks!